specs/src/specs/namespace.md

```diff
@@ -94,7 +94,8 @@ Among the potential consequences is the _Woods Attack_, as elaborated in this fo
 
 ## Implementation
 
-See [go-square/namespace](https://github.com/celestiaorg/go-square/blob/main/shares/namespace.go).
+See the [namespace implementation in go-square](https://github.com/celestiaorg/go-square/blob/be3c2801e902a0f90f694c062b9c4e6a7e01154e/namespace/namespace.go). 
```

Comment:
The permalink provided here is based on [this file](https://github.com/celestiaorg/go-square/blob/v1.1.0/namespace/namespace.go) from v1.1.0 of go-square.